mediacapture-fromelement/cross-origin.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+</head>
+<body>
+<video autoplay controls id="output"></video>
+<script>
+
+// Run captureStream() on cross-origin <video> content
+  async_test(function() {
+    const video = document.createElement('video');
+    video.src = location.origin.replace("//", "//www1.") + "/media/test-v-128k-320x240-24fps-8kfr.webm";
```

Comment:
test-v-128k-320x240-24fps-8kfr.webm is about 38KB in size. If we don't care about the contents, we could consider using white.webm which is just 10KB, and will perhaps avoid flakes in slow bots.